packages/esm-outpatient-app/src/visit-header/visit-header.scss

```diff
@@ -4,7 +4,7 @@
 
 .topNavHeader {
   top: var(--omrs-offline-banner-height);
-  @include brand-02(background-color);
+  @include brand-01(background-color);
```

Comment:
@donaldkibet why did you change this back to brand-01, it is supposed to be brand-02 ```#004144``` according to mockups and what Ciaran mentioned

Reply:
No, in particular, we can correct it

Reply:
Okay, great, I will put it back to brand-02, it needs to be different from the normal NavBar.